Nucleus stops work.

Nucleus-0.8.0-S5-1.10.2-plugin
spongeforge-1.10.2-2073-5.0.0-BETA-1687
```
[21:08:32] [Server thread/INFO] [nucleus/sponge]: Nucleus is now entering the post-init phase.
[21:08:32] [Server thread/ERROR] [Sponge/sponge]: Could not pass FMLPostInitializationEvent to Plugin{id=nucleus, name=Nucleus, version=0.8.0-5.0, description=The Ultimate Essentials Plugin., source=E:\sponge1.10.2\mods\Nucleus-0.8.0-S5-1.10.2-plugin.jar}
java.lang.NoClassDefFoundError: org/spongepowered/api/data/translator/ConfigurateTranslator
	at io.github.nucleuspowered.nucleus.configurate.typeserialisers.ItemStackSnapshotSerialiser.deserialize(ItemStackSnapshotSerialiser.java:49) ~[ItemStackSnapshotSerialiser.class:0.8.0-5.0]
	at io.github.nucleuspowered.nucleus.configurate.typeserialisers.ItemStackSnapshotSerialiser.deserialize(ItemStackSnapshotSerialiser.java:21) ~[ItemStackSnapshotSerialiser.class:0.8.0-5.0]
	at ninja.leaping.configurate.objectmapping.serialize.TypeSerializers$ListSerializer.deserialize(TypeSerializers.java:213) ~[spongeforge-1.10.2-2073-5.0.0-BETA-1687.jar:1.10.2-2073-5.0.0-BETA-1687]
	at ninja.leaping.configurate.objectmapping.serialize.TypeSerializers$ListSerializer.deserialize(TypeSerializers.java:199) ~[spongeforge-1.10.2-2073-5.0.0-BETA-1687.jar:1.10.2-2073-5.0.0-BETA-1687]
	at ninja.leaping.configurate.objectmapping.ObjectMapper$FieldData.deserializeFrom(ObjectMapper.java:80) ~[spongeforge-1.10.2-2073-5.0.0-BETA-1687.jar:1.10.2-2073-5.0.0-BETA-1687]
	at ninja.leaping.configurate.objectmapping.ObjectMapper$BoundInstance.populate(ObjectMapper.java:140) ~[spongeforge-1.10.2-2073-5.0.0-BETA-1687.jar:1.10.2-2073-5.0.0-BETA-1687]
	at ninja.leaping.configurate.objectmapping.serialize.TypeSerializers$AnnotatedObjectSerializer.deserialize(TypeSerializers.java:243) ~[spongeforge-1.10.2-2073-5.0.0-BETA-1687.jar:1.10.2-2073-5.0.0-BETA-1687]
	at ninja.leaping.configurate.SimpleConfigurationNode.getValue(SimpleConfigurationNode.java:252) ~[spongeforge-1.10.2-2073-5.0.0-BETA-1687.jar:1.10.2-2073-5.0.0-BETA-1687]
	at io.github.nucleuspowered.nucleus.dataservices.dataproviders.ConfigurateDataProvider.load(ConfigurateDataProvider.java:42) ~[ConfigurateDataProvider.class:0.8.0-5.0]
	at io.github.nucleuspowered.nucleus.dataservices.Service.load(Service.java:27) ~[Service.class:0.8.0-5.0]
	at io.github.nucleuspowered.nucleus.Nucleus.onPostInit(Nucleus.java:177) ~[Nucleus.class:0.8.0-5.0]
	at org.spongepowered.common.event.listener.GamePostInitializationEventListener_Nucleus_onPostInit12.handle(Unknown Source) ~[?:?]
	at org.spongepowered.common.event.RegisteredListener.handle(RegisteredListener.java:95) ~[RegisteredListener.class:1.10.2-2073-5.0.0-BETA-1687]
	at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:349) [SpongeModEventManager.class:1.10.2-2073-5.0.0-BETA-1687]
	at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:390) [SpongeModEventManager.class:1.10.2-2073-5.0.0-BETA-1687]
	at org.spongepowered.mod.SpongeMod.onStateEvent(SpongeMod.java:215) [SpongeMod.class:1.10.2-2073-5.0.0-BETA-1687]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_101]
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) [minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) [minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) [minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) [minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:275) [minecraft_server.1.10.2.jar:?]
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:239) [LoadController.class:?]
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:217) [LoadController.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_101]
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) [minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) [minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) [minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) [minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:275) [minecraft_server.1.10.2.jar:?]
	at net.minecraftforge.fml.common.LoadController.redirect$onPost$0(LoadController.java:53) [LoadController.class:?]
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:142) [LoadController.class:?]
	at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:793) [Loader.class:?]
	at net.minecraftforge.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:107) [FMLServerHandler.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:333) [FMLCommonHandler.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:214) [ld.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:431) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_101]
Caused by: java.lang.ClassNotFoundException: org.spongepowered.api.data.translator.ConfigurateTranslator
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_101]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_101]
	... 44 more
Caused by: java.lang.NullPointerException
```

--- a/src/main/java/io/github/nucleuspowered/nucleus/configurate/typeserialisers/ItemStackSnapshotSerialiser.java
+++ b/src/main/java/io/github/nucleuspowered/nucleus/configurate/typeserialisers/ItemStackSnapshotSerialiser.java
@@ -12,7 +12,7 @@ import ninja.leaping.configurate.objectmapping.serialize.TypeSerializer;
 import org.slf4j.Logger;
 import org.spongepowered.api.Sponge;
 import org.spongepowered.api.data.DataContainer;
-import org.spongepowered.api.data.translator.ConfigurateTranslator;
+import org.spongepowered.api.data.persistence.DataTranslators;
 import org.spongepowered.api.item.inventory.ItemStackSnapshot;
 
 import java.util.List;
@@ -28,7 +28,8 @@ public class ItemStackSnapshotSerialiser implements TypeSerializer<ItemStackSnap
 
     @Override
     public ItemStackSnapshot deserialize(TypeToken<?> type, ConfigurationNode value) throws ObjectMappingException {
-        // Process enchantments, temporary fix before Sponge gets a more general fix in.
+        // Process enchantments, temporary fix before Sponge gets a more general
+        // fix in.
         ConfigurationNode ench = value.getNode("UnsafeData", "ench");
         if (!ench.isVirtual()) {
             List<? extends ConfigurationNode> enchantments = ench.getChildrenList();
@@ -46,12 +47,13 @@ public class ItemStackSnapshotSerialiser implements TypeSerializer<ItemStackSnap
         }
 
         Optional<ItemStackSnapshot> oiss = Sponge.getDataManager().deserialize(ItemStackSnapshot.class,
-                ConfigurateTranslator.instance().translateFrom(value));
+                DataTranslators.CONFIGURATION_NODE.translate(value));
         if (oiss.isPresent()) {
             return oiss.get();
         }
 
-        // If we get here, we have had a problem with the data. We should therefore remove all the data
+        // If we get here, we have had a problem with the data. We should
+        // therefore remove all the data
         // (except enchants) and see what happens.
         ConfigurationNode unsafe = value.getNode("UnsafeData");
         unsafe.getChildrenMap().forEach((k, v) -> {
@@ -61,7 +63,7 @@ public class ItemStackSnapshotSerialiser implements TypeSerializer<ItemStackSnap
         });
 
         // Try again
-        oiss = Sponge.getDataManager().deserialize(ItemStackSnapshot.class, ConfigurateTranslator.instance().translateFrom(value));
+        oiss = Sponge.getDataManager().deserialize(ItemStackSnapshot.class, DataTranslators.CONFIGURATION_NODE.translate(value));
         if (oiss.isPresent()) {
             logger.warn(Util.getMessageWithFormat("config.itemstacksnapshot.data", value.getNode("ItemType").getString()));
             return oiss.get();
@@ -76,6 +78,6 @@ public class ItemStackSnapshotSerialiser implements TypeSerializer<ItemStackSnap
     @Override
     public void serialize(TypeToken<?> type, ItemStackSnapshot obj, ConfigurationNode value) throws ObjectMappingException {
         DataContainer container = obj.toContainer();
-        value.setValue(ConfigurateTranslator.instance().translateData(container));
+        value.setValue(DataTranslators.CONFIGURATION_NODE.translate(container));
     }
 }

--- a/src/main/java/io/github/nucleuspowered/nucleus/modules/vanish/commands/VanishCommand.java
+++ b/src/main/java/io/github/nucleuspowered/nucleus/modules/vanish/commands/VanishCommand.java
@@ -65,7 +65,7 @@ public class VanishCommand extends CommandBase<CommandSource> {
         boolean toVanish = args.<Boolean>getOne(b).orElse(!playerToVanish.get(Keys.INVISIBLE).orElse(false));
 
         DataTransactionResult dtr = playerToVanish.offer(Keys.VANISH, toVanish);
-        playerToVanish.offer(Keys.VANISH_IGNORES_COLLISION, toVanish);
+        playerToVanish.offer(Keys.VANISH_PREVENTS_TARGETING, toVanish);
         playerToVanish.offer(Keys.VANISH_IGNORES_COLLISION, toVanish);
         playerToVanish.offer(Keys.IS_SILENT, toVanish);
 

